feat(recording): add recording button/indicator

frankemax opened the pull request:
closes: https://github.com/mconf/bbb-mobile-sdk/issues/414
closes: https://github.com/mconf/bbb-mobile-sdk/issues/426
closes: https://github.com/mconf/bbb-mobile-sdk/issues/447
closes: https://github.com/mconf/bbb-mobile-sdk/issues/450
closes: https://github.com/mconf/bbb-mobile-sdk/issues/451

germanocaumo committed:
feat(recording): add recording button/indicator

germanocaumo committed:
feat(recording): redo according to design

frankemax commented:
Portal needs to handle the record permission @matheus-lagreca

germanocaumo commented:
@frankemax it's missing the second commit of the PR (with the design implementation)

frankemax committed:
Merge remote-tracking branch 'origin/v0.8.z-release' into 414-port-record-session

frankemax committed:
fix(custom-drawer): adjust button styles

frankemax committed:
feat(notification): beautify message design

frankemax committed:
fix(recording): avoid NaN label

frankemax committed:
refactor(record): modify icon color

frankemax commented:
> @frankemax it's missing the second commit of the PR (with the design implementation)

@germanocaumo Whoops... thanks!

frankemax committed:
feat(recording-button): add pulse animation

frankemax committed:
fix(recording-indicator): add missing parameter

frankemax committed:
feat(recording-indicator): update styles to match figma

Now the size is correct
Stops from sliding horizontally according to screen name

frankemax committed:
Merge remote-tracking branch 'origin/v0.8.z-release' into 414-port-record-session

frankemax committed:
feat(recording): remove button from custom drawer

Move it all to header button, and controls to inside modal

Lucas-Schell committed:
feat(custom-data): handle custom record

Add predicate and listener to handle addUserSettings makeCall

Lucas-Schell committed:
feat(user-settings): add redux slice

frankemax committed:
feat(record-icon): remove pause icon

frankemax committed:
refactor(user-settings): remove dependency cycle
